python/taichi/lang/expr.py

```diff
@@ -46,51 +47,61 @@ def stack_info():
         # remove the confusing last line
         return '\n'.join(raw.split('\n')[:-3]) + '\n'
 
+    def __neg__(self):
```

Comment:
I thought you are already inheriting from `TaichiOperations`? Do we still need to override these?

Reply:
Forgot to delete, sorry!